src/services/mondomarket/index.jsx

```
@@ -0,0 +1,238 @@
+              </p>
+            )}
+            <p className="text-gray-700 mt-4">{mondo?.description}</p>
+          </div>
+
+          <ProfileCard image={mondo?.userimage} title={mondo?.username} />
+
+          <div className="flex items-center justify-center gap-10 mb-5 mt-8 cursor-pointer">
+            <div
+              className="flex flex-col justify-center items-center"
+              onClick={() => setOfferModal(true)}
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)